src/helpers/data.js

```diff
@@ -105,31 +105,37 @@ export default {
       .map((datum) => {
         const x = accessor.x(datum);
         const y = accessor.y(datum);
-        return merge({}, datum, {
-          category: this.determineCategoryIndex(x, props.categories),
+        const category = this.determineCategoryIndex(x, props.categories);
+
+        return assign(
```

Comment:
Seeing `TypeError: 'undefined' is not a function (evaluating 'Object.assign')`.  Investigating whether `Object.assign` support is included in `builder-victory-component`.